docs/cli/cli-data.md

```diff
@@ -326,25 +324,212 @@ that are of standard (aka not test) quality. Therefore, these filters can be eas
 planet data filter --permission --std-quality --asset ortho_analytic_8b_sr | planet data search PSScene --filter -
 ```
 
-## `data asset` command basics
```

Comment:
On line 293, suggest changing:

which can be interested to inspect

to:

which can be interesting to inspect